.githooks/pre-push

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh
+
+# This hook is called with the following parameters:
+#
+# $1 -- Name of the remote to which the push is being done
+# $2 -- URL to which the push is being done
+#
+# If pushing without using a named remote those arguments will be equal.
+#
+# Information about the commits which are being pushed is supplied as lines to
+# the standard input in the form:
+#
+#   <local ref> <local oid> <remote ref> <remote oid>
+#
```

Comment:
Did we mean to include the githook in the public repo?

Reply:
Yes, for the willing. The _daring_ even :) At least it's optional.